packages/hawtio/src/plugins/camel/CamelTreeView.tsx

```diff
@@ -1,21 +1,30 @@
 import React, { ChangeEvent, useState, useContext } from 'react'
-import { PluginTreeViewToolbar, MBeanNode, MBeanTree } from '@hawtio/plugins/shared'
+import { PluginTreeViewToolbar, MBeanNode, MBeanTree } from '@hawtiosrc/plugins/shared'
 import { TreeView, TreeViewDataItem } from '@patternfly/react-core'
 import { CamelContext } from './context'
 
+/*
+ * Expansion requires more than 2 states since the expandAll
+ * must be removed completely to defer to the expanded state
+ * of each data node
+ */
+enum ExpansionValue {
+  Default = 0, // should revert to the expanded state of the data
```

Comment:
these line comments can be jsdoc as well. and I think the value assignment is superfluous for enums.